etc/portage/sets/eessi-2022.11-linux-x86_64

```diff
@@ -0,0 +1,19 @@
+app-editors/emacs
+app-editors/vim
+app-shells/bash-completion
+dev-python/GitPython
+dev-python/keyring
+dev-python/pip
+dev-python/pycodestyle
+dev-python/pyyaml
+dev-python/rich
+dev-util/patchelf
+media-fonts/dejavu
+media-fonts/liberation-fonts
+=sys-apps/archspec-0.1.3::eessi
```

Comment:
Latest version if 0.1.4, we should update to that?

Reply:
Release notes seem not to indicate that it would add a lot more (https://github.com/archspec/archspec/releases), but sure can be updated.

Reply:
Updated anyhow.